### ISSUE TITLE
ng add @angular/material is not working

### ISSUE THREAD

**Issue by gautamkrishnar** [angular/material2#12000]
#### Bug, feature request, or proposal:
ng add @angular/material is not working

#### What is the expected behavior?
Must work

#### What is the current behavior?
ng add @angular/material is showing
```bash
Collection "@angular/material" cannot be resolved.
Error: Collection "@angular/material" cannot be resolved.
    at NodeModulesEngineHost._resolveCollectionPath (/Users/gautamkrishnar/WebstormProjects/app-source/app-source/node_modules/@angular-devkit/schematics/tools/node-module-engine-host.js:88:15)
    at NodeModulesEngineHost.createCollectionDescription (/Users/gautamkrishnar/WebstormProjects/app-source/app-source/node_modules/@angular-devkit/schematics/tools/file-system-engine-host-base.js:106:27)
    at SchematicEngine._createCollectionDescription (/Users/gautamkrishnar/WebstormProjects/app-source/app-source/node_modules/@angular-devkit/schematics/src/engine/engine.js:78:40)
    at SchematicEngine.createCollection (/Users/gautamkrishnar/WebstormProjects/app-source/app-source/node_modules/@angular-devkit/schematics/src/engine/engine.js:71:43)
    at Object.getCollection (/Users/gautamkrishnar/WebstormProjects/app-source/app-source/node_modules/@angular/cli/utilities/schematics.js:28:31)
    at AddCommand.getOptions (/Users/gautamkrishnar/WebstormProjects/app-source/app-source/node_modules/@angular/cli/models/schematic-command.js:193:41)
    at AddCommand.<anonymous> (/Users/gautamkrishnar/WebstormProjects/app-source/app-source/node_modules/@angular/cli/commands/add.js:30:49)
    at Generator.next (<anonymous>)
    at /Users/gautamkrishnar/WebstormProjects/app-source/app-source/node_modules/@angular/cli/commands/add.js:7:71
    at new Promise (<anonymous>)
```

**Comment by gautamkrishnar** [comment]
Installed a previous version. Looks like current version is broken:
````bash
 ng add @angular/material@6.2.1
```

**Comment by josephperrott** [comment]
Closing as this was fixed by #11839 and will go out in todays release.

**Comment by gautamkrishnar** [comment]
Thanks 👍
